src/util.ts

```diff
@@ -55,3 +58,10 @@ export const promiseAny = async <T>(
     (value) => Promise.resolve<T>(value)
   );
 };
+
+// Format User-Agent: <product> / <product-version> (<platform>)
+// source: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
+export const getUserAgent = (): string => {
+  const platformName = os.platform();
+  return `sigstore-js/${packageJson.version} (${platformName})`;
```

Comment:
Thoughts on also including the node version and architecture alongside the platform? Not sure it needs to be blocking, could be a follow up maybe?

Reply:
not sure if we need to account for non-node envs like deno

Reply:
sounds good! i can add it as my next pr :)